workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -255,3 +253,20 @@ async def map_fid_to_uid(dataset: str, feature_id: str, fid_field: str) -> str:
     async with get_connection() as conn:
         record = await conn.fetchrow(query, feature_id)
     return str(record[0])
+
+
+async def get_shdi(bpoly: Union[Polygon, MultiPolygon]) -> float:
+    """Get Subnational Human Development Index (SHDI) for a bounding polygon.
+
+    Get SHDI by intersecting the bounding polygon with sub-national regions provided by
+    the GlobalDataLab (GDL).
+
+    If intersection with multiple GDL regions occurs, return the sum of SHDI in
```

Comment:
I think it's not the "sum in proportion of the intersection area" but should be the "weighted average (using area as the weight)"

Reply:
Yes, using your formulation is much clearer. I corrected the docstring.